Utilities/AzureOnboardingCLI/azure-automatic-role-assignment.py

```diff
@@ -68,7 +106,7 @@ def run_command(cmd, *args):
         Exception: If the command fails to run, an exception is raised with the command and error message.
 
     Example:
-        >>> run_command("ls", "-l")
+        #>>> run_command("ls", "-l")
```

Comment:
?

Reply:
pycharm has a big problem with this statement without the comment i have no idea why